src/commons/BrowserUtils.ts

```diff
@@ -847,9 +879,9 @@ export namespace BrowserUtils {
   /**
    * @param mouseButton -  {LEFT = 0, MIDDLE = 1 , RIGHT = 2}
    */
-  export function releaseMouseButton(mouseButton: string): void {
+  export function releaseMouseButton(mouseButton: number): void {
     //Defaults to the left mouse button if not specified.
-    const selectedMouseButton: string =
+    const selectedMouseButton: number =
       mouseButton === undefined ? MouseButton.LEFT : mouseButton;
     Reporter.step(`Release mouse button '${selectedMouseButton}'`);
     browser.buttonUp(Number(selectedMouseButton));
```

Comment:
@Ad1B3nAr0ya I think you can change `Number(selectedMouseButton)` to `selectedMouseButton` now that you've changed it from String to number

Reply:
changed